### PR TITLE
Re-add set_date_livraison as a deprecated function.

### PR DESCRIPTION
**Pull request by fappels**
Add deprecation phase, think used in quite some external or custom modules.

Also use correct naming convention for new method naming.

### CODE REVIEW

File: htdocs/comm/propal/class/propal.class.php [hunk 1]
```diff
@@ -2060,16 +2060,30 @@ public function set_echeance($user, $date_fin_validite, $notrigger = 0)
...
 	 *
 	 *	@param      User 	$user        		Object user that modify
 	 *	@param      int		$delivery_date     Delivery date
 	 *  @param  	int		$notrigger			1=Does not execute triggers, 0= execute triggers
 	 *	@return     int         				<0 if ko, >0 if ok
 	 */
-    public function set_delivery_date($user, $delivery_date, $notrigger = 0)
+    public function setDeliveryDate($user, $delivery_date, $notrigger = 0)
 	{
         // phpcs:enable
```
Comment (frederic34) [inline htdocs/comm/propal/class/propal.class.php:2088]: @fappels can you move all "phpcs enable" in set_date_livraison ?

File: htdocs/delivery/class/delivery.class.php [hunk 2]
```diff
@@ -1007,7 +1007,7 @@ public function getRemainingDelivered()
 	 *	@param      integer 		$delivery_date     Delivery date
 	 *	@return     int         						<0 if KO, >0 if OK
 	 */
-    public function set_delivery_date($user, $delivery_date)
+    public function setDeliveryDate($user, $delivery_date)
 	{
         // phpcs:enable
```
Comment (frederic34) [inline htdocs/delivery/class/delivery.class.php:1012]: here can delete the two phpcs disable enable

Comment (fappels) [reply htdocs/delivery/class/delivery.class.php]: Done, thanks Frederic.

Comment (frederic34) [reply htdocs/delivery/class/delivery.class.php]: :+1:

**Pull request merged**